Lib/test/mp_preload_flush.py

```diff
@@ -0,0 +1,14 @@
+import multiprocessing
+import sys
+
+modname = 'preloaded_module'
+if __name__ == '__main__':
+    assert modname not in sys.modules
+    multiprocessing.set_start_method('forkserver')
+    multiprocessing.set_forkserver_preload([modname])
+    for _ in range(2):
+        p = multiprocessing.Process()
+        p.start()
+        p.join()
+else:
+    assert modname in sys.modules
```

Comment:
To make sure that the test is also run when python -O is used:

```suggestion
elif modname not in sys.modules:
    raise AssertionError(f'{modname!r} is not in sys.modules')
```

Reply:
Ah, good point, thanks!